sound/soc/sof/intel/mtl.c

```diff
@@ -431,6 +431,10 @@ int mtl_dsp_cl_init(struct snd_sof_dev *sdev, int stream_tag, bool imr_boot)
 
 	dev_dbg(sdev->dev, "Primary core power up successful\n");
 
+	/* set primary core mask and refcount to 1 */
+	sdev->enabled_cores_mask = BIT(SOF_DSP_PRIMARY_CORE);
+	sdev->dsp_core_ref_count[SOF_DSP_PRIMARY_CORE] = 1;
```

Comment:
commit message: set-up primary core info on MeteorLake platform

Reply:
Why isn't this done in mtl_dsp_core_power_up()

Reply:
good idea. Thanks!